core-web/libs/dotcms-scss/angular/dotcms-theme/components/form/_inputtext.scss

```diff
@@ -0,0 +1,53 @@
+@use "variables" as *;
+@import "common";
+
+.p-inputtext {
+    @extend #input-text-extend;
+
+    &.p-inputtext-sm {
+        @extend #input-text-sm;
+    }
+
+    &.p-inputtextarea {
+        @extend #input-text-area;
+    }
+
+    &.p-dropdown-label,
+    &.p-autocomplete-multiple-container {
+        height: unset; // Some components use input text underneath but they have their own height
+        color: $black;
+    }
+
+    &.p-dropdown-filter,
+    &.p-autocomplete-input {
+        color: $black;
+    }
+}
+
+.p-input-icon-right {
+    i {
+        color: $color-palette-primary;
+        cursor: pointer;
+        right: $spacing-1;
+    }
+    &:has(.p-inputtext.p-inputtext-sm) {
+        i {
+            font-size: $font-size-sm;
+            // 1 rem is the main font size, so we need to calculate the difference between the main font size and the small font size
+            // and then subtract that from the top margin (setted by primeNG) to keep the icon vertically centered on small version
```

Comment:
We need to find a way to do this responsive, flex or some trick, but definitely not calculation here.

Reply:
I'm not sure about it, because the icon is inside the input, that's why we need to use relative/absolute